.platform.app.yaml

```diff
@@ -42,16 +44,14 @@ relationships:
     database: "database:mysql"
 
 web:
-    locations:
-        '/':
-            # The public directory of the application relative to its root.
-            root: 'web'
-            # The front-controller script which determines where to send
-            # non-static requests.
-            passthru: '/index.php'
```

Comment:
This is how we move from a PHP-FPM setup to a ReactPHP setup on Platform.sh (documentation: https://docs.platform.sh/languages/php.html#alternate-start-commands)